### ISSUE TITLE
`pull` docs are outdated, not reflecting the breaking change

### ISSUE THREAD

**Issue by secondmanveran** [EndBug/add-and-commit#354]
**Describe the bug**
Error: Error: fatal: 'NO-PULL' does not appear to be a git repository

**Workflow used**
```yaml
      - name: commit assets to branch
        uses: EndBug/add-and-commit@v8
        with:
          default_author: github_actor
          new_branch: latest
          message: 'chore(deploy): deploy main code to production'
          add: '-f public/'
          pull: 'NO-PULL'
          push: '-f origin latest'
```

**Expected behavior**
I expect it to work. Your docs say clearly to enter 'NO_PULL' as an option if you don't want the action to pull. Now suddenly with this new version it breaks.

**Logs**
https://github.com/jetstreamlabs/jetstreampro/runs/4914677888?check_suite_focus=true

**Comment by EndBug** [comment]
Hi, thanks for opening the issue!

This is a breaking change of version 8.0.0, but I forgot to add it to the README and the CHANGELOG... 😅 
I've only updated the action manifest, [here](https://github.com/EndBug/add-and-commit/blob/v8.0.0/action.yml#L42).

Anyway, you can simply remove the `pull` input entirely and the action won't pull, this is the new default behavior.
I'll try to fix the docs as soon as possible!

**Comment by EndBug** [comment]
@all-contributors please add @secondmanveran for their docs contribution

**Comment by EndBug** [comment]
This fix has been published in `v8.0.1` (also `v8` and `latest`). Thanks again for your contribution ❤️
